src/backend/api/Fusion.Resources.Logic/Workflows/InternalRequestDirectWorkflowV1.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Fusion.Resources.Database.Entities;
+using System.Collections.Generic;
+
+namespace Fusion.Resources.Logic.Workflows
+{
+
+    public class InternalRequestDirectWorkflowV1 : WorkflowDefinition
+    {
+        public const string CREATED = "created";
+        public const string PROVISIONING = "provisioning";
+
+        public override string Version => "v1";
+        public override string Name => "Direct personnel assignment request";
+
+        public InternalRequestDirectWorkflowV1()
+            : base(null)
+        {
+            Steps = new List<WorkflowStep>()
+            {
+                Created,
+                Provisioning
+            };
+        }
+
+        public InternalRequestDirectWorkflowV1(DbPerson creator)
+            : this()
+        {
+            Step(CREATED)
+                .SetName("Approved")
```

Comment:
Not sure this should be "Approved" ?. When there is no approval step, sounds a bit wierd. Maybe just have "Created" as thats the step thats done?